playground/backend/cmd/server/controller.go

```diff
@@ -309,3 +309,15 @@ func (controller *playgroundController) GetPrecompiledObjectLogs(ctx context.Con
 	response := pb.GetPrecompiledObjectLogsResponse{Output: logs}
 	return &response, nil
 }
+
+// GetPrecompiledObjectGraph returns the graph of the compiled and run example
+func (controller *playgroundController) GetPrecompiledObjectGraph(ctx context.Context, info *pb.GetPrecompiledObjectGraphRequest) (*pb.GetPrecompiledObjectGraphResponse, error) {
+	cd := cloud_bucket.New()
```

Comment:
Why is it called _cd_, not a _cb_ for example?

Reply:
Done.